book/A-git-in-other-environments/sections/atom.asc

```diff
@@ -0,0 +1,23 @@
+=== Git in Atom
+
+(((Atom)))
+Atom ships with a built-in package called GitHub, which provides a fairly-complete interface to Git operations and a deep integration with the website GitHub.
+There are different ways to access the package, probably the most common way is through the keybinding:
+
+Open the Git panel: `Ctrl+9`
+
+Another way is from the menu: `Packages -> GitHub -> Toggle Git Tab`
+
+GitHub comes with plenty of documentation when it comes to the integration with a remote repository on GitHub, which you can find within atom "Flight Manual".
```

Comment:
Rationale for change:
- Atom is a noun and so needs to be capitalized. 
- Also we need a 's to indicate possession. Whose flight manual is it? It is Atom's Flight Manual.
---
Suggested situation after implementing changes:

GitHub comes with plenty of documentation when it comes to the integration with a remote repository on GitHub, which you can find within **Atom's** "Flight Manual".